scanner/src/test/kotlin/ScanResultsStorageTest.kt

```diff
@@ -127,49 +116,44 @@ class ScanResultsStorageTest : WordSpec({
                 storageReaders = listOf("cd", "file")
             )
 
-            val storage = configureStorage(config)
-
-            storage.readers shouldHaveSize 2
+            val storage = ScanResultsStorage.configure(config)
+            storage should beInstanceOf<CompositeStorage>()
+            (storage as CompositeStorage).readers shouldHaveSize 2
             val cdStorage = storage.readers[0]
             val fileStorage = storage.readers[1]
-            cdStorage.shouldBeInstanceOf<ClearlyDefinedStorage>()
-            fileStorage.shouldBeInstanceOf<FileBasedStorage>()
+            cdStorage should beInstanceOf<ClearlyDefinedStorage>()
+            fileStorage should beInstanceOf<FileBasedStorage>()
             storage.writers shouldContainExactly listOf(fileStorage, cdStorage)
         }
 
         "detect references to unknown storage readers" {
-            val ex = shouldThrow<IllegalArgumentException> {
+            val exception = shouldThrow<IllegalArgumentException> {
                 val config = ScannerConfiguration(
-                    storages = mapOf("reader" to ClearlyDefinedStorageConfiguration("http://url")),
+                    storages = mapOf(
+                        "cdProd" to ClearlyDefinedStorageConfiguration("http://prod.cd"),
+                        "cdDev" to ClearlyDefinedStorageConfiguration("http://dev.cd")
+                    ),
                     storageReaders = listOf("nonExistingReader")
                 )
-                configureStorage(config)
+                ScanResultsStorage.configure(config) should beInstanceOf<CompositeStorage>()
```

Comment:
I find it a bit confusing that you added a check here. The configure() function is expected to throw an exception, so the instanceOf check will never be executed. And if it was executed and failed, the resulting failure message would be misleading.